docs/resources/mws_log_delivery.md

```diff
@@ -0,0 +1,36 @@
+# databricks_mws_log_delivery Resource
+
+-> **Note** This resource has evolving API, which may change in future versions of provider.
+
+This resource configures delivery of various log types from databricks workspaces. Backend does not support removal of log delivery configuration, so provider handles it through simply disabling the config.
+
+## Example Usage
+
+```hcl
+resource "databricks_mws_customer_managed_keys" "my_cmk" {
+    account_id   = var.account_id
+    credentials_id = databricks_mws_credentials.log_writer.credentials_id
+    storage_configuration_id = databricks_mws_storage_configurations.log_bucket.storage_configuration_id
+    config_name = "Usage Logs"
+    log_type = "BILLABLE_USAGE"
+    output_format = "CSV"
+}
+```
+
+## Argument reference
```

Comment:
can we refer to public open api docs in case this changes in future.

Reply:
I'm not sure if cloud engineers / devops folks would be reading any other documentation than https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/cluster, but i'll put a link to docs.